### PR TITLE
changes tequila sunrise recipe's sugar to grenadine, and lowers the alcohol content of many tall drinks (and adds a period to a typo'd drink description)

### PR DESCRIPTION
**Pull request by JordantheKitty**

### CODE REVIEW

File: code/modules/chemistry/Reagents-FoodDrink.dm [hunk 1]
```diff
@@ -943,7 +943,7 @@ datum
 			fluid_r = 99
 			fluid_g = 32
 			fluid_b = 15
-			alch_strength = 0.15 //adding milk shouldn't quadruple the alcohol per volume
+			alch_strength = 0.2 //adding milk shouldn't quadruple the alcohol per volume
```
Comment (BatElite) [inline code/modules/chemistry/Reagents-FoodDrink.dm:946]: This one's an increase, is that intentional?

Comment (JordantheKitty) [reply code/modules/chemistry/Reagents-FoodDrink.dm:946]: oh it was a decrease to .15 but then i increased it from that up to .2

Comment (JordantheKitty) [reply code/modules/chemistry/Reagents-FoodDrink.dm:946]: it was a decrease to .15 but then i upped it a bit back to .2

**Comment by UrsulaMejor** [comment]
Seems fine to me. Good first pr.

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**